### DISCUSSION TITLE
Deploy snap but inspect source shows different code to that deployed

### DISCUSSION THREAD

**Discussion by javaspeak** [MetaMask/snaps#2082] [Q&A]
I carry out these steps:

(i) deploy the snap into metamask flask 
(ii) click the 3 dots in metamask and choose snaps
(iii) click on my snap
(iv) Click on the Version link => shows me "Index of snap"
(v)  Move into dist folder and open bundle.js
(vi) Inside bundle.js I search for unique string used in the snap insight and I find it:

"No Signal Available AAAAB"

(vii) I then click Send in Flask and transfer some MATIC from Account 1 to Account 2
(viii) This triggers the insight - when I click on the insight I see:

"No Signal Available"

If I inspect the snap I also see "No Signal Available" instead of "No Signal Available AAAAB"

It is like the snap is showing some old cached version instead of the deployed code.

Even if right click on Chrome's Refresh button and choose "Empty Cache and Hard Reload" I still get the old version.

Any ideas?

**Comment by javaspeak** [comment]
Does the snap cache its code somewhere different to the browser cache? - even emptying everything in the cache in advanced mode is not working.

**Comment by FrederikBolding** [comment]
@javaspeak Is this with a local snap or a snap installed from NPM?
